packages/math-input/src/components/input/mathquill-instance.ts

```diff
@@ -0,0 +1,5 @@
+import MathQuill from "mathquill";
+
+import {MathQuillInterface} from "../../types";
+
+export default MathQuill.getInterface(2) as MathQuillInterface;
```

Comment:
This isn't the MathQuill input (MathField), this is basically the MathQuill library/API. We use it to make a MathField, but we also use constants that are stuck to it.